src/structures/APIMessage.js

```diff
@@ -249,7 +249,7 @@ class APIMessage {
   /**
    * Resolves a single file into an object sendable to the API.
    * @param {BufferResolvable|Stream|FileOptions|MessageAttachment} fileLike Something that could be resolved to a file
-   * @returns {Object}
+   * @returns {APIAttachment}
```

Comment:
This method returns an object with the properties `attachment`, `name`, and `file`, that doesn't align with `APIAttachment`.

Reply:
Should I make a typedef instead or is this somewhere else on the API docs?

Reply:
This is some object of our own, so if anything, a typedef.

Reply:
Alright, I’ll change that

Reply:
Resolved in the latest commit, please check it out